src/main/java/uk/gov/hmcts/darts/dailylist/controller/DailyListController.java

```diff
@@ -51,6 +58,10 @@ public class DailyListController implements DailyListsApi {
     private DailyListProcessor processor;
 
     @Override
+    @SecurityRequirement(name = SECURITY_SCHEMES_BEARER_AUTH)
+    @Authorisation(contextId = ANY_ENTITY_ID,
+        securityRoles = {},
```

Comment:
Will you be removing the empty `securityRoles = {},` now that we have a default in the annotation?